### PR TITLE
feat: Change the space menu options in the spaces page - MEED-1491 - Meeds-io/meeds#549 

### PR DESCRIPTION
**Pull request by MayTekayaa**
This fix will change the space menu options in the spaces page.

### CODE REVIEW

File: webapp/portlet/src/main/webapp/vue-apps/spaces-list/components/ExoSpaceCardFront.vue [hunk 1]
```diff
@@ -97,20 +163,24 @@
         </a>
       </div>
 
-      <v-card-text class="spaceCardBody align-center pt-2 pb-1">
+      <v-card-text 
+        class="spaceCardBody align-center"
+        :class="!isMobile && 'pt-2 pb-1'">
```
Comment (boubaker) [inline webapp/portlet/src/main/webapp/vue-apps/spaces-list/components/ExoSpaceCardFront.vue]:
```suggestion
        class="spaceCardBody align-center pt-2 pb-1 py-sm-0">
```

File: webapp/portlet/src/main/webapp/vue-apps/spaces-list/components/ExoSpaceCardFront.vue [hunk 2]
```diff
@@ -97,20 +163,24 @@
...
-          class="spaceDisplayName text-truncate d-block">
+          :class="isMobile && 'text-truncate-2 mt-0' || 'text-truncate d-block'"
+          class="spaceDisplayName">
           {{ space.displayName }}
         </a>
-        <v-card-subtitle
+        <a 
+          :href="url"
+          :class="isMobile && 'my-0'"
           class="spaceMembersLabel py-0">
```
Comment (boubaker) [inline webapp/portlet/src/main/webapp/vue-apps/spaces-list/components/ExoSpaceCardFront.vue]:
```suggestion
          class="spaceMembersLabel py-0 my-0 my-sm-auto">
```

Comment (MayTekayaa) [reply webapp/portlet/src/main/webapp/vue-apps/spaces-list/components/ExoSpaceCardFront.vue]: done!

File: webapp/portlet/src/main/webapp/vue-apps/spaces-list/components/ExoSpaceCardFront.vue (hunk 1 above)
Comment (MayTekayaa) [reply webapp/portlet/src/main/webapp/vue-apps/spaces-list/components/ExoSpaceCardFront.vue]: done!

File: webapp/portlet/src/main/webapp/vue-apps/spaces-list/components/ExoSpaceCardFront.vue [hunk 3]
```diff
@@ -26,19 +26,85 @@
         </v-btn>
         <v-spacer />
         <exo-space-favorite-action
-          v-if="space.isMember && favoritesSpaceEnabled"
+          v-if="space.isMember && favoritesSpaceEnabled && !isMobile"
           :is-favorite="space.isFavorite"
           :space-id="space.id" />
+        <div v-else-if="isMobile">
```
Comment (boubaker) [inline webapp/portlet/src/main/webapp/vue-apps/spaces-list/components/ExoSpaceCardFront.vue:32]: Please dissociate both conditions, it's not related and thus shouldn't use `v-else-if` here.

```suggestion
        <div v-if="isMobile">
```

File: webapp/portlet/src/main/webapp/vue-apps/spaces-list/components/ExoSpaceCardFront.vue [hunk 4]
```diff
@@ -277,6 +347,15 @@ export default {
         return '#';
       }
     },
+    isMobile() {
+      return this.$vuetify.breakpoint.mdAndDown;
```
Comment (boubaker) [inline webapp/portlet/src/main/webapp/vue-apps/spaces-list/components/ExoSpaceCardFront.vue:351]: Mobile = `xsOnly`

```suggestion
      return this.$vuetify.breakpoint.xsOnly;
```

**Pull request closed without merging**